packages/timeline-state-resolver/src/integrations/tricaster/triCasterStateDiffer.ts

```diff
@@ -143,14 +155,18 @@ export class TriCasterStateDiffer {
 			matrixOutputs: this.matrixOutputNames,
 		}
 
-		this.timelineStateConverter = new TriCasterTimelineStateConverter(() => this.getDefaultState(), resourceNames)
+		this.timelineStateConverter = new TriCasterTimelineStateConverter(
```

Comment:
Hmm.. since the class we are in is called `TriCasterStateDiffer` I wouldn't expect it to also convert stuff into TriCaster state. I would expect it to just receive TriCaster states and diff them. 
My point being the conversion of timelineObject to TriCaster state is not something the classing checking for differences in TriCaster state should know about.

Reply:
I agree, but I'll leave it like this until the next refactor